docker/nginx/nginx-key.conf

```diff
@@ -116,6 +116,12 @@ http {
             proxy_pass http://opcua-adapter:${MF_OPCUA_ADAPTER_HTTP_PORT};
         }
 
+        # Proxy pass to mainflux-twins
+        location /twins {
```

Comment:
location ~ ^/(twins|states) {

as the states of the twins also have to be loaded from the twins service